fulltext/backends/__ocr.py

```diff
@@ -25,23 +30,29 @@
 
 def read(img, **kargs):
     lang = kargs.get('lang', 'eng')
-    rotate = kargs.get('rotate', 0)
+    rotate = kargs.get('rotate', None)
 
     im = Image.open(img)
 
-    if not rotate:
+    if rotate is None:
         try:
             exif = im._getexif()
+
         except AttributeError:
             # No EXIF data, no rotation necessary.
             pass
+
         else:
             rotate = EXIF_ROTATION.get(exif.get(EXIF_ORIENTATION, None), 0)
 
     if rotate:
         im = im.rotate(rotate)
 
-    return pytesseract.image_to_string(im, lang=lang)
+    try:
+        return pytesseract.image_to_string(im, lang=lang)
+
+    except FileNotFoundError:
```

Comment:
I think this will break on Python 2.

Reply:
The tests pass, but the linter does not. I will fix that up.